File: java/Payroll/Person.java
public class Person {
    private String firstName;
    private String lastName;
    private BankAccount bankAccount;
    private float payPerHour;
    private float taxRate;
    public static float OVERTIME_MULTIPLIER = 1.5f;

    public Person(String firstName, String lastName, BankAccount bankAccount, float payPerHour, float taxRate) {
        this.firstName = firstName;
        this.lastName = lastName;
        this.bankAccount = bankAccount;
        this.payPerHour = payPerHour;
    }

    public String getFullName() {
        return firstName + " " + lastName;
    }

    public float getPayPerHour() {
        return this.payPerHour;
    }
}

File: python/payouts/person.py
import uuid


class Person:
    first_name: str
    last_name: str
    id: uuid

    def __init__(self, first_name: str, last_name: str):
        self.first_name = first_name
        self.last_name = last_name
        self.uuid = uuid.uuid4()

    def __repr__(self):
        return f"{self.first_name} {self.last_name}"
